src/ploigos_step_runner/utils/file.py

```diff
@@ -156,3 +156,28 @@ def create_parent_dir(file_path):
     parent_dir_path = os.path.dirname(file_path)
     if parent_dir_path:
         os.makedirs(parent_dir_path, exist_ok=True)
+
+def base64_encode(file_path):
+    """Given a file_path, read and encode the contents in base64
+    Returns
+    -------
+    Base64Contents
+        base64 encoded string of file contents
+    """
+    encoding = Path(file_path).read_text().encode('utf-8')
+    return base64.b64encode(encoding).decode('utf-8')
+
+def get_file_hash(file_path):
```

Comment:
need to add unit tests to the https://github.com/ploigos/ploigos-step-runner/blob/main/tests/utils/test_file.py class for this function.

a good test to make sure you have good per class unit test coverage is to run the tests just for that class and makre sure you get 100% coverage for the tested class.

For example:

`python3 -m pytest --cov --cov-report term-missing tests/utils/test_file.py`